tensorflow_quantum/python/layers/circuit_executors/expectation_test.py

```diff
@@ -192,28 +275,41 @@ def test_expectation_simple_tf_train(self):
         self.assertAllClose(mse.numpy(), 0, atol=1e-3)
 
 
-class ExpectationFunctionalTests(tf.test.TestCase):
+class ExpectationFunctionalTests(parameterized.TestCase, tf.test.TestCase):
     """Test hybrid/integrated models that include an expectation layer."""
 
-    def test_simple_param_value_input(self):
+    @parameterized.parameters([
+        {
+            'backend': 'noisy'
+        },
+        {
+            'backend': None  # old API usage
```

Comment:
Should the `cirq.Simulator()` backend also be added to these parameterized tests?  Not sure about adding to one test and not the others

Reply:
I looked through the history and this was originally the only test we had that used the simulator and I think I know why: After adding a `cirq.Simulator` case to all the other tests it tripled the runtime of the entire files tests so I think we had originally struck a balance between coverage and runtime, that might be worth keeping.